panic on windows

GOROOT=D:\Go #gosetup
GOPATH=D:\gopath #gosetup
D:\Go\bin\go.exe build -o C:\Users\Administrator\AppData\Local\Temp\___compile.exe D:\gopath\src\github.com\756445638\lucy\src\cmd\compile\main.go #gosetup
"D:\Program Files\JetBrains\GoLand 2017.3\bin\runnerw.exe" C:\Users\Administrator\AppData\Local\Temp\___compile.exe #gosetup
panic: open /dev/urandom: The system cannot find the path specified.

goroutine 1 [running]:
github.com/timtadh/data-structures/set.init.0()
	D:/gopath/src/github.com/timtadh/data-structures/set/sortedset.go:18 +0x119
github.com/timtadh/data-structures/set.init()
	<autogenerated>:1 +0x78
github.com/timtadh/data-structures/linked.init()
	<autogenerated>:1 +0x69
github.com/timtadh/lexmachine/dfa.init()
	<autogenerated>:1 +0x5f
github.com/timtadh/lexmachine.init()
	<autogenerated>:1 +0x55
github.com/756445638/lucy/src/cmd/compile/lex.init()
	<autogenerated>:1 +0x57
github.com/756445638/lucy/src/cmd/compile/parser.init()
	<autogenerated>:1 +0x5c
github.com/756445638/lucy/src/cmd/compile/lc.init()
	<autogenerated>:1 +0x6b
main.init()
	<autogenerated>:1 +0x50

Process finished with exit code 2



seems on windows the file "/dev/urandom" dose not exist !!!
this library can not be used on windows??

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -3,9 +3,9 @@
 
 [[projects]]
   name = "github.com/timtadh/data-structures"
-  packages = ["errors","hashtable","linked","list","set","test","tree","tree/avl","types"]
-  revision = "d69ddac42beae463c5770d2b2b8a4b75acbe556a"
-  version = "v0.5.1"
+  packages = ["errors","hashtable","linked","list","rand","set","test","tree","tree/avl","types"]
+  revision = "cd0c1de5390876f92ce0762c9c626a569b124f24"
+  version = "v0.5.2"
 
 [[projects]]
   name = "github.com/timtadh/getopt"
@@ -16,6 +16,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "58b4fb3066fe44929915ef093cc60a40b24fd4de80545ee5a95b6c1cacec6939"
+  inputs-digest = "76403af9b8313eb4c8dec248b6daa284bd8bf99ca2aebb7c22bd1b5ad203cb65"
   solver-name = "gps-cdcl"
   solver-version = 1

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -14,4 +14,4 @@ ignored = []
 ## dep whether coming from the Gopkg.toml of the current project or a dependency.
 [[constraint]]
 name = "github.com/timtadh/data-structures"
-version = ">=0.5.1"
+version = ">=0.5.2"

